template/register.html

```diff
@@ -0,0 +1,15 @@
+{% block title %}Register Here{% endblock %}
+
+<div> 
```

Comment:
you can create a separate template for the messages and then import it at the very top level of your html